### PR TITLE
feat: prometheus rule

### PR DESCRIPTION
**Pull request by omerlh**

### CODE REVIEW

File: charts/dafka-consumer/templates/prometheus-rule.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- with .Values.prometheusRule.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ .Values.name | default .Release.Name }}-kafka-consumer
+spec:
+  groups:
+    - name: Ledge Global Rules
```
Comment (guysegal) [inline charts/dafka-consumer/templates/prometheus-rule.yaml:12]: name to "Rules"

**Review by guysegal — COMMENTED** [review]
.

File: charts/dafka-consumer/README.md [hunk 2]
```diff
@@ -65,6 +65,8 @@ A Helm Chart for Dafka Consumer
 | podMonitor.sampleLimit | string | `nil` | set sample limit for the pod monitor |
 | pdb.enabled | bool | `false` | Set to true to enable |
 | jvmOpts | string | `nil` |  |
+| prometheusRule.enabled | bool | `false` |  |
```
Comment (guysegal) [inline charts/dafka-consumer/README.md:68]: prometheusRules (plural)

**Review by guysegal — APPROVED** [review]

File: charts/dafka-consumer/README.md (hunk 2 above)
Comment (omerlh) [reply charts/dafka-consumer/README.md:68]: No because it one object that can have many groups

**Pull request merged**